packages/web-components/src/utils/table.spec.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx React.createElement */
```

Comment:
similar to the comment I left on the table sort PR, I wonder if this file should have a more specific name. Yes, these functions are specific to the table component, but we could be more specific here and call it something like `table-migration` maybe.